src/main/java/mate/academy/util/InitDatabase.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.util;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import mate.academy.exception.DataProcessingException;
+
+public class InitDatabase {
+    public static void executeInitScript(Connection connection) {
+        String scriptPath = "src/main/resources/init_db.sql";
+        try {
+            String sql = new String(Files.readAllBytes(Paths.get(scriptPath)));
```

Comment:
It's not necessary to read the entire SQL script into a single String. Consider reading and executing each statement separately to handle large scripts more efficiently.